src/TMap.cpp

```diff
@@ -2200,7 +2200,8 @@ void TMap::downloadMap(const QString& remoteUrl, const QString& localFileName)
     // Attempts to ensure INFO message gets shown before download is initiated!
 
     pHost->updateProxySettings(mpNetworkAccessManager);
-    mpNetworkReply = mpNetworkAccessManager->get(QNetworkRequest(QUrl(url)));
+    mudlet::self()->setNetworkRequestDefaults(url, request);
```

Comment:
Shouldn't this replace Lines 2182-2193 above? And shouldn't that request object be used below instead of creating a new one and ignoring the one we just built?

Reply:
Fixed.